packages/data-table-server/src/dataTableService/services/SurveyResponseDataTableService.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { TupaiaApiClient } from '@tupaia/api-client';
+import { yup } from '@tupaia/utils';
+import { Country, Entity, Survey, SurveyResponse } from '@tupaia/types';
+import { DataTableService } from '../DataTableService';
+import { getDefaultEndDate, getDefaultStartDate } from './utils';
+
+const requiredParamsSchema = yup.object().shape({
+  ids: yup.array().of(yup.string()).default([]),
+  assessorNames: yup.array().of(yup.string()).default([]),
+  countryCodes: yup.array().of(yup.string()).default([]),
+  surveyCodes: yup.array().of(yup.string()).default([]),
+  entityCodes: yup.array().of(yup.string()).default([]),
+  startDate: yup.date().default(getDefaultStartDate),
+  endDate: yup.date().default(getDefaultEndDate),
```

Comment:
I'd probably lean towards no default values for start/end date, feels a little unexpected that we'd be filtering by default